pkg/testcase/pod.go

```diff
@@ -57,3 +71,11 @@ func processPodStatus(
 		}
 	}
 }
+
+func processCiliumStatus(pod shared.Pod) {
```

Comment:
I'm stuck here because it will still return a PASS based on this code if BOTH pods are in Pending, whereas it should only return PASS if ONE pod is in Pending and the other one is Running. Can we update this to somehow reflect that instead?

Reply:
good catch, i was not counting on that